react-backend/routes/export.js

```diff
@@ -0,0 +1,97 @@
+var express = require("express");
+var router = express.Router();
+var mysql = require("mysql");
+var config = require("../config");
+var bodyParser = require("body-parser");
+const fastcsv = require("fast-csv");
+const fs = require("fs");
+
+const connection = mysql.createPool({
+  host: config.host,
+  user: config.user,
+  password: config.password,
+  database: config.database
+});
+
+/* GET users listing. */
+router.get("/:associatedAccount", function(req, res, next) {
+  var account = req.params.associatedAccount;
+
+  if (!account) {
+    console.log("associatedAccount not defined");
+    res.send("associatedAccount not defined");
+  } else {
+    connection.getConnection(function(err, connection) {
+      // Executing the MySQL query (select all data from the 'users' table).
+      connection.query(
+        "select * from Transaction where associatedAccount =" + account,
+        function(error, results, fields) {
+          // If some error occurs, we throw an error.
+          if (error) throw error;
+
+          // Getting the 'response' from the database and sending it to our route. This is were the data is.
+          const jsonData = JSON.parse(JSON.stringify(results));
+          console.log("jsonData", jsonData);
+          const ws = fs.createWriteStream("transactions_fastcsv.csv");
+
+          fastcsv
+          .write(jsonData, { headers: true })
+          .on("finish", function() {
+              res.send("Exported transactions to transactions_fastcsv.csv successfully!");
+            })
+            .pipe(ws);
+        }
+      );
+    });
+  }
+});
+
+router.get("/:associatedAccount/:otherColumns", function(req, res, next) {
```

Comment:
I've done absolutely no testing on this router, but if it worked for you, it works for me!

Reply:
I'm just going to remove it.